src/components/Charts/ModelChart.js

```diff
@@ -310,6 +311,17 @@ const ModelChart = ({
     <ChartContainer>
       <Wrapper projections={projections} isInferred={projection.isInferred}>
         <Chart options={options} />
+        <Outcomes
+          title="Predicted Outcomes"
+          projections={[projections.baseline, projections.primary]}
+          colors={[
```

Comment:
huh the colors + the projections + the intervention? seems like we could just pass in projections object

Reply:
So I ditched intervention, but kept projections / colors, since the logic for which ones to show should match the logic for the charts above, so it seems better to keep it in one place, though _really_ we should share the logic, but unfortunately that's not straightforward to do with the way things are set up right this moment.